### PR TITLE
A function has been added to remove special characters from the phone

### PR DESCRIPTION
**Pull request by AnnaLangner**
The function of removing special characters from the phone has been added

### CODE REVIEW

File: script/main.py [hunk 1]
```diff
@@ -45,6 +55,19 @@ def create_new_record_with_dob_in_json(records):
     record["dob"].update(dob_new_record_time_until_birthday)
 
 
+def removes_special_characters_from_phone_and_cell_numbers(phone_records):
+  for record in phone_records:
+    phone = record['phone']       
+    clear_phone = re.findall(r'[0-9]', phone)
```
Comment (klangner) [inline script/main.py]: This can be simplify with function str.replace()

File: script/main.py [hunk 2]
```diff
@@ -45,6 +55,19 @@ def create_new_record_with_dob_in_json(records):
...
+    clear_phone = ''.join(clear_phone)
+    record['phone'] = clear_phone
+
+    cell = record['cell'] 
+    clear_cell = re.findall(r'[0-9]', cell)
```
Comment (klangner) [inline script/main.py]: Some here

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/main.py [hunk 3]
```diff
@@ -22,27 +22,53 @@ def init_argparser():
...
+        date_of_the_nearest_birthday = date_of_the_nearest_birthday.replace(day=29)      
+    days_left = abs(date_of_the_nearest_birthday - today).days    
+    dob_new_record_time_until_birthday = {"time_until_birthday": days_left}
     record["dob"].update(dob_new_record_time_until_birthday)
+  
+
+def removes_special_characters_from_phone_and_cell_numbers(phone_records):
+  for record in phone_records:
+    phone = record['phone']       
+    clear_phone = phone.replace("(", "").replace(")", "").replace("-", "").replace("+", "").replace(" ", "")
```
Comment (klangner) [inline script/main.py]: You can use regex here

**Review by klangner — CHANGES_REQUESTED** [review]

File: script/main.py (hunk 3 above)
Comment (AnnaLangner) [reply script/main.py]: I don't know how to use "str.replace ()" and "regex" together

Comment (klangner) [reply script/main.py]: check:
https://stackoverflow.com/questions/11475885/python-replace-regex

Comment (AnnaLangner) [reply script/main.py]: I've seen it before, but it doesn't mean anything to me. Here they recommended using "re.sub", which I do in my code.

Comment (AnnaLangner) [reply script/main.py]: When I changed the code to this:
```
  phone = record['phone']       
    clear_phone = phone.replace('\(|\)|\-|\+|\s', '')
    record['phone'] = clear_phone

    cell = record['cell'] 
    clear_cell = cell.replace('\(|\)|\-|\+|\s', '')
    record['cell'] = clear_cell
``` 
Nothing happens :\

Comment (klangner) [reply script/main.py]: You are not using re.sub here.
Check the code:
```
re.sub(r'a', 'b', 'banana')
```
How can you modify it to get what you want?
Also regarding the regex itself. You are only interested in digits. How to write regex to replace all not digit characters?

Comment (AnnaLangner) [reply script/main.py]: In my last changes I have this:

```
phone = record['phone']       
    clear_phone = re.sub(r'\(|\)|\-|\+|\s', '', phone)
    record['phone'] = clear_phone

    cell = record['cell'] 
    clear_cell = re.sub(r'\(|\)|\-|\+|\s', '', cell)
    record['cell'] = clear_cell
```

Comment (klangner) [reply script/main.py]: Ok. this is correct. But you can improve it in the following way
You want only numbers from the phone number, yes? So to match any character which is not number you can use the following regex expression
```
clear_phone = re.sub(r'\D', '', phone)
```

**Review by klangner — APPROVED** [review]

**Pull request merged**